assets/app/scripts/services/applicationGenerator.js

```diff
@@ -231,8 +222,8 @@ angular.module("openshiftConsole")
           },
           source: {
             git: {
-              ref: sourceRef,
-              uri: sourceUrl
+              ref: input.buildConfig.gitRef || 'master',
```

Comment:
I'll add it back. What if git ref is both in the URL and a different value is specified in the input?